cypress/support/cypress-commands/assertion.js

```diff
@@ -158,16 +158,14 @@ Cypress.Commands.add(
       cy.validateEvent(extractedApiObject, context, validationPath, expected, appId);
     } else {
       const apiResponseContent = eval(CONSTANTS.EXTRACTEDAPI_PATH + validationPath);
-      const pretext = CONSTANTS.METHOD_CONTENT;
-      cy.log(
-        pretext +
-          ' expected ' +
-          JSON.stringify(apiResponseContent) +
-          ' to be ' +
-          JSON.stringify(expected)
-      ).then(() => {
-        assert.deepEqual(apiResponseContent, expected, pretext);
-      });
+      const pretext =
+        CONSTANTS.METHOD_CONTENT +
+        ' expected ' +
+        JSON.stringify(apiResponseContent) +
+        ' to be ' +
+        JSON.stringify(expected);
+      // Executing fireLog.deepEqual() after logging
+      fireLog.deepEqual(apiResponseContent, expected, pretext);
```

Comment:
Do we need pretext here? Doesn't firelog already create a message?
Can't this just be
`fireLog.deepEqual(apiResponseContent, expected, CONSTANTS.METHOD_CONTENT);`

Also remove the comment `// Executing fireLog.deepEqual() after logging`

Reply:
Pretext can be removed, but removing it changes the log from 

[5/10/2024 11:23:57 AM]["Method Content validation expected \"1395767607303387913\" to be \"1395767607303387913\""]

to 

[5/10/2024 11:25:36 AM]["1395767607303387913"]